db/schemas/schemas.py

```diff
@@ -23,7 +24,7 @@ class Meta:
 
     workflow_status = Enum(Status)
     language = Enum(Language)
-
+    flags = Nested("FlagMetadata", many=True)
```

Comment:
Is it possible to add a test for this to understand how it is affecting the JSON response?

Reply:
Yes, I will add testing